disnake/guild.py

```diff
@@ -4233,13 +4233,36 @@ async def search_members(
             resp.append(member)
         return resp
 
+    @overload
+    async def get_or_fetch_members(
+        self,
+        user_ids: List[int],
+        *,
+        presences: bool = ...,
+        cache: bool = ...,
+        as_dict: Literal[False] = ...,
+    ) -> List[Member]:
+        ...
+
+    @overload
+    async def get_or_fetch_members(
+        self,
+        user_ids: List[int],
+        *,
+        presences: bool = ...,
+        cache: bool = ...,
+        as_dict: Literal[True] = ...,
```

Comment:
```suggestion
        presences: bool = False,
        cache: bool = True,
        as_dict: Literal[False] = False,
    ) -> List[Member]:
        ...

    @overload
    async def get_or_fetch_members(
        self,
        user_ids: List[int],
        *,
        presences: bool = False,
        cache: bool = True,
        as_dict: Literal[True],
```
Not specifying a default in the second overload makes them non-ambiguous

Reply:
https://github.com/DisnakeDev/disnake/pull/977/commits/4bdffdd1fec5b4366a7f36a56bbe1723e3743a3e